use-cases/credit_risk/risk_bucketing.ipynb

```diff
@@ -0,0 +1,932 @@
+{
```

Comment:
<p>How are you determining that the optimal number is 2 when the three different methods suggest 4, 2, and 5? How should customers compare the results of these different techniques?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
I've added markup explaining to explain this.